packages/fether-react/src/BackupPhrase/BackupPhrase.js

```diff
@@ -0,0 +1,399 @@
+      event.preventDefault();
+
+      this.setState({
+        error: error.message + '. \n Please check your password and try again.'
+      });
+
+      return false;
+    }
+  };
+
+  render () {
+    const {
+      account: { address, name },
+      location: { pathname }
+    } = this.props;
+    const { error, unlocked } = this.state;
+
+    const needsRewrite = pathname.split('/')[3] === 'true';
+
+    const title = needsRewrite
+      ? 'Backup Recovery Phrase'
+      : 'View Recovery Phrase';
+
+    return (
+      <React.Fragment>
+        <RequireHealthOverlay require='node'>
+          <Header
+            left={
+              <Link className='icon -back' to='/accounts'>
+                Back
+              </Link>
+            }
+            title={<h1>{title}</h1>}
+          />
+
+          <div className='window_content'>
+            <div className='box -padded'>
+              <AccountCard
+                address={address}
+                name={name}
+                drawers={[
+                  <div>
+                    {unlocked
+                      ? this.renderCopyAndRewrite()
+                      : this.renderPasswordForm()}
+                    {needsRewrite && unlocked ? this.renderTips() : null}
+                    {unlocked && this.renderButtons()}
+                    {error}
+                  </div>
+                ]}
+              />
+            </div>
+          </div>
+          <nav className='footer-nav'>
+            <div className='footer-nav_status'>
+              <Health />
+            </div>
+          </nav>
+        </RequireHealthOverlay>
+      </React.Fragment>
+    );
+  }
+
+  renderButtons () {
+    const { phrase, phraseRewrite, step } = this.state;
+    const {
+      location: { pathname }
+    } = this.props;
+
+    const needsRewrite = pathname.split('/')[3] === 'true';
+
+    return (
+      <nav className='form-nav -space-around'>
+        <button className='button -back' onClick={this.prevStep} type='button'>
+          Back
+        </button>
+        {step === 1 ? (
+          <button autoFocus className='button' onClick={this.nextStep}>
+            {needsRewrite ? 'Next' : 'Done'}
+          </button>
+        ) : (
+          <button
+            autoFocus
+            className='button'
+            disabled={phraseRewrite !== phrase}
+          >
+            Confirm
+          </button>
+        )}
+      </nav>
+    );
+  }
+
+  renderCopyAndRewrite () {
+    const { step } = this.state;
+
+    if (step === 1) {
+      return this.renderCopyForm();
+    } else {
+      return this.renderRewriteForm();
+    }
+  }
+
+  renderCopyForm () {
+    const { phrase } = this.state;
+    const {
+      location: { pathname }
+    } = this.props;
+
+    const needsRewrite = pathname.split('/')[3] === 'true';
+
+    return (
+      <div>
+        <div className='text -centered'>
+          {needsRewrite
+            ? 'Please write your recovery phrase on a piece of paper.'
```

Comment:
To avoid a useless scroll bar
```suggestion
            ? 'Write down your recovery phrase'
```